src/matlab.ts

```diff
@@ -10,26 +10,34 @@ import * as script from "./script";
 
 export interface Version {
```

Comment:
I typically think of a release having a version, not a version having a release. All releases have a version, not all versions are releases.

```
Release
- has a Name
- has an Update number
- has a Version, and Version can be converted to a Semantic Version
```

Reply:
This makes sense. For now Version and Semantic Version aren't really distinct, but we might need them to be at some point. Ideally we won't have to track it at that level indefinitely and update our releaseInfo file for each update